lib/rules/accessor-pairs.js

```diff
@@ -63,3 +63,11 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.documentation = {
+    errors: ["Getter is not present", "Setter is not present"],
```

Comment:
Agreed. If it really becomes a problem, we could add a test step that checks errors generated by the tests against the messages supplied in `meta.errors`.